@here/harp-examples/README.md

```diff
@@ -22,42 +22,42 @@ We can group the examples by several categories, looking at what kind of feature
 
 ### Getting Started
 
-1. [The modular hello world example](./src/getting-started_hello-world_npm.ts) that displays a map in an HTML element with our default map style and default data source, using NPM packages.
-1. [The HTML hello world example](./src/getting-started_hello-world_js-bundle.ts) similar to the former but showing how to use harp in an HTML page with inline JS scripts.
-1. [Globe projection](./src/getting-started_globe-projection.ts) shows the setup to use a globe projection.
-1. [Camera with free movement](./src/getting-started_free-camera.ts), that demonstrates how the camera works in `harp.gl`.
-1. [Orbit camera](./src/getting-started_orbiting-view.ts), showcases the `lookAt` method and how to use it to orbit around a point.
-1. [Themes examples](./src/getting-started_open-sourced-themes.ts) features the various themes open sourced with Harp.
+1. [The modular hello world example](https://github.com/heremaps/harp.gl/blob/master/%40here/harp-examples/src/getting-started_hello-world_npm.ts) that displays a map in an HTML element with our default map style and default data source, using NPM packages.
```

Comment:
Might be better to link to the example instead of the source file b/c from the example you can show the source but from the source file there is no easy way to find the example

Reply:
Do you mean linking to e.g. https://www.harp.gl/docs/master/examples/#getting-started_hello-world_npm.html instead of https://github.com/..... ?